paper.tex

```diff
@@ -149,48 +151,31 @@ \subsubsection{Research data management}
-This allocated money can then be utilised to delegate/dispatch a central RSE person or group into a research project for a few weeks or months as necessary.
-
-The HPC community,
-arguably a sub-community of the RSE-community,
+The field of RSE teams in Germany is currently poorly mapped and very diverse.
+It includes groups with responsibilities across institutions like the Helmholtz HIFIS group\footnote{\url{https://events.hifis.net/category/4/}}\autocite{haupt_hifis_consulting_2021},
+but mostly groups with responsibilities local to their institution.
+Their host within institutions also differs.
+Some like the Scientific Software Center in Heidelberg\footnote{\url{https://www.ssc.uni-heidelberg.de/en}}\autocite{ulusoy_heidelberg_ssc_2024} are part of an interdisciplinary center, goverend directly researchers.
+Others like the Competence Center Digital Research (zedif) in Jena\footnote{\url{https://www.zedif.uni-jena.de/en/}} are part of the central university research support infrastructure and are goverend by the institutional leadership.
+Yet others have a project character, like Suresoft in Braunschweig\footnote{\url{https://suresoft.dev/}}~\autocite{Blech2022}.
+Another national pioneer is the Göttingen State and University Library where RSE services are part of the library service portfolio.
+
+Most of these groups, or at least their RSE services, are still relatively young.
+However, some groups can already report a remarkable increase in software quality, better grant applications, less brain drain and overall better employee satisfaction levels~\autocite{schimavoigt2023}.
+RSE services which benefit all disciplines/departments may represent a unique selling point for organisations competing for the brightest minds.
+
+Besides institutions, there are also other communities related to RSE\@.
+The HPC community, arguably a sub-community of the RSE-community,
 has a history of offering training and consulting to prospective and active users of the hardware they support.
 National Competence Centres\footnote{EuroCC: \url{https://www.eurocc-access.eu/}} form a network of HPC-RSE consulting groups to share expertise with academic and industry actors\autocite{eurocc_success_stories_2023,eurocc_success_stories_2024}.
 
+While we focus on Germany in this paper, it is beneficial to review how other countries approach research software.
+In the UK, which we choose as example for being the oldest in terms of RSE, many universities started initiating dedicated RSE units about a decade ago~\autocite{Crouch2013, KDL}.
+The successful establishment of these units is a role model for similar research performing organisations worldwide.
+A range of already-existing RSE units in the UK can be seen on this (incomplete) map: \url{https://society-rse.org/community/rse-groups/}.
+In the UK, almost all grant applications include software development in their budget.
```

Comment:
That claim sounds wrong. Pretty sure there is grant applications without software aspects. Also, I know that RSE departments also failed at some UK universities. Furthermore, I know researchers from the UK that develop software without an RSE department.